examples/cylinder_internal_pressure/cylinder_internal_pressure.jl

```diff
@@ -105,11 +106,14 @@ function solve(::FirstCase)
     # Structural Analysis
     # -------------------------------
     s = structure(liner_elastic)
-    sa = LinearStaticAnalysis(s; NSTEPS)
+    # TODO: Revise possible singular K matrix influenced by LinearSolver initialization.
```

Comment:
I found some instability here, anyway with default `LinearSolve.jl` the test still passes.